README.md

```diff
@@ -1,31 +1,22 @@
 
 # Ansible JupyterHub Server with Prometheus Stacks for STFC cloud Openstacks
-Provides a Magnum Kubernetes cluster, with autoscaling enabled and configured,
-and a JupyterHub Service. This uses the helm chart provided by [ZeroToJupyterHub](https://github.com/jupyterhub/zero-to-jupyterhub-k8s).
+Provides a JupyterHub Service on an existing Openstack Cluster. This uses the helm chart provided by [ZeroToJupyterHub](https://github.com/jupyterhub/zero-to-jupyterhub-k8s).
 ## Contents
 - [Features](#features)
 - [Limitations](#limitations)
 - [Requirements](#requirements)
   * [Local Environment Setup](#local-environment-setup)
-  * [Recommended Setup](#recommended-setup)
-- [Deploying a cluster](#deploying-a-cluster)
-  * [Variables (`/playbooks/deploy_cluster.yml`)](#variables----playbooks-deploy-clusteryml--)
-  * [Instruction](#instruction)
 - [GPU](#gpu)
-  * [Variables (`/playbooks/build_gpu_driver.yml`)](#variables----playbooks-build-gpu-driveryml--)
-  * [Enabling GPU Workers](#enabling-gpu-workers)
-  * [Manual Cluster config fix for GPU Workers](#manual-cluster-config-fix-for-gpu-workers)
-  * [Priority Class for GPU-operator](#priority-class-for-gpu-operator)
 - [Kubectl Namespaces](#kubectl-namespaces)
 - [Jupyter Hub Config](#jupyter-hub-config)
   * [HTTPS Config](#https-config)
     + [Setting up DNS for Lets Encrypt](#setting-up-dns-for-lets-encrypt)
     + [Using existing TLS Certificate](#using-existing-tls-certificate)
-  * [Using Oauth Sign in (`config-oauth.yaml.template`)](#using-oauth-sign-in---config-oauthyamltemplate--)
-    + [Setting users / admin groups for Oauth](#setting-users---admin-groups-for-oauth)
-  * [Using Native Authenticator (`config-native.yaml.template`)](#using-native-authenticator---config-nativeyamltemplate--)
+  * [Using Oauth Sign in (`config-oauth.yaml.template`)](#using-oauth-sign-in-config-oauthyamltemplate)
```

Comment:
(We need to remove this section as it's not really supported of tested, but for a future PR)